scrapy/utils/asyncio.py

```
@@ -0,0 +1,23 @@
+def install_asyncio_reactor():
+    """ Tries to install AsyncioSelectorReactor
+    """
+    try:
```

Could you refactor these `except: pass` ocurrences to use [`contextlib.suppress`](https://docs.python.org/3/library/contextlib.html#contextlib.suppress)?

---

@elacuesta one of these cases uses `else` specifically to handle the situation when there is no `ImportError`, can that still be refactored?

---

I don’t think so, although using a `return` instead of `pass` in this specific case may be _slightly_ more readable.

---

Right, I missed that. I agree, `return` instead of `pass` in the first occurrence does sound better to me.